src/main/java/com/google/cloud/tools/managedcloudsdk/command/CommandCaller.java

```diff
@@ -69,7 +69,19 @@ public String call(
       }
       return stdOutSaver.getResult().get();
     } catch (IOException | ExecutionException ex) {
-      throw new CommandExecutionException(ex);
+      String stdOut;
+      String stdErr;
+      try {
+        stdOut = stdOutSaver.getResult().get();
+      } catch (InterruptedException | ExecutionException ignored) {
+        stdOut = "stdout collection interrupted";
+      }
+      try {
+        stdErr = stdErrSaver.getResult().get();
+      } catch (InterruptedException | ExecutionException ignored) {
+        stdErr = "stderr collection interrupted";
+      }
+      throw new CommandExecutionException(stdOut + "\n" + stdErr, ex);
```

Comment:
in CommandExitException, error is stored in a separate field instead of the actual `message` field of the exception. I feel that's cleaner. I'd also suggest that stdout and stderr be kept  in 2 different fields and let the caller render it.

Reply:
done